pcsx2/GS/Renderers/HW/GSRendererNew.cpp

```diff
@@ -741,7 +738,7 @@ void GSRendererNew::EmulateBlending(bool& DATE_GL42, bool& DATE_GL45)
 
 		// Require the fix alpha vlaue
 		if (ALPHA.C == 2)
-			m_conf.cb_ps.alpha_fix = ALPHA.FIX;
+			m_conf.cb_ps.TA_MaxDepth_Af.a = ALPHA.FIX;
```

Comment:
```suggestion
			m_conf.cb_ps.TA_MaxDepth_Af.a = ALPHA.FIX / 128.f;
```
Old was int, new is float